### PR TITLE
[IMP] inventory: update create second warehouse doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] targeting the `create_a_second_warehouse` doc in Odoo's Inventory app documentation, specific to 17.0.

Here's what I did:
- Rewrote/updated copy and sections for accuracy specific to 17.0
- Replaced screenshots/added new screenshots with 17.0 UI
- Added admonitions and expanded doc to be more robust

[Project Task Link](https://www.odoo.com/web#cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form&id=3900418)

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hi @Felicious, this PR is ready for content review when you have a chance. Thanks in advance!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 1]
```diff
@@ -2,99 +2,135 @@
...
+- :guilabel:`Dropship Subcontractors`: dropship subcontractors with components.
+- :guilabel:`Resupply Subcontractors`: allow subcontractors to be resupplied with goods from this
+  warehouse.
+- :guilabel:`Manufacture to Resupply`: when products are manufactured, they can be manufactured in
+  this warehouse.
+- :guilabel:`Manufacture`: the route that should be followed when manufacturing goods inside
+  the warehouse. Choose from one, two, and three-step production.
+- :guilabel:`Buy to Resupply`: when products are purchased, they can be delivered to this warehouse.
+- :guilabel:`Resupply From`: tick the checkbox next to the warehouse that this warehouse should be
+  resupplied from. Resupply routes are automatically created for the selected warehouse.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:76]: I'm not familiar with the resupply from another warehouse route, so this section was confusing to me. Given that the resupply route is only available as a field when there's more than one warehouse created in a database, I think it's important to expand on this field a little more.

(For example, when I only have 1 WH, the "Resupply From" field isn't there:
image
 )
 
 Can we possibly:
 - elaborate a little more what a "warehouse that this warehouse should be resupplied from", means?
   - e.g. "Check the box next to the **other** warehouses from which **this** warehouse can replenish products. If the current warehouse does not have the stock, it can also pull products from these selected warehouses in the database. Automatic resupply routes will be created accordingly."
 - link to the [resupply WH doc](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.html) for more info

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 2]
```diff
@@ -2,99 +2,135 @@
...
-- :guilabel:`Counted Quantities`: choose whether the counted quantities for products being added
-  should default to stock on hand or zero; does not affect inventory adjustments for new warehouses
-  since they have no existing inventory
-
-.. image:: create_a_second_warehouse/inventory-adjustment-configuration.png
-   :align: center
-   :alt: A filled out form for an inventory adjustment.
+If a new warehouse is created with existing inventory in it, the inventory count should be added to
+Odoo, so the stock listed in the Odoo database reflects the real stock count in the physical
+warehouse.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:89]: The current explanation does a good job of explaining what we need to do in Odoo, but we can reword it to elaborate on *why* we need to adjust the stock in the first place.

```suggestion
If a new warehouse is created to represent a real-world storage location with existing stock, update the warehouse's stock using an inventory adjustment.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 3]
```diff
@@ -2,99 +2,135 @@
...
+Doing so adds a new line at the bottom of the list of inventory adjustments.
 
-- :guilabel:`Product`: the product being added to inventory
-- :guilabel:`Location`: the location where the product is currently stored in the new warehouse;
-  this can be set as the overall warehouse or a location within the warehouse
+Configure the line by filling out the following fields as necessary:
+
+- :guilabel:`Location`: the location where the product is currently stored in the new warehouse.
+  This can be set as the overall warehouse, or a location within the warehouse.
+- :guilabel:`Product`: the product being added to inventory.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:99]: This section on defining the fields in the Inventory adjustment page is really helpful! However, it seems like we might have some duplicate information in our documentation, since the [Inventory Adjustments doc](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.html) already covers this in great detail. Maybe we could streamline it a bit to avoid redundancy? 

We can either replace this and the following lines with a "seealso" admonition block linking to the Inventory Adjustments page, and/or summarize what we can do on this page.

**Review by Felicious — APPROVED** [review]
Hi @brse-odoo ! Another well-written doc, as always! :blush: 

Your definition of the warehouse fields are spot-on, and it's awesome to see you updating this doc! Until now, these warehouse configuration details haven't received much attention, so your work today is truly appreciated! 😄 (The "Warehouse and Locations" doc can also use your attention, if you're up for it :eyes: )

II've got a few suggestions for areas that could benefit from a bit more elaboration. They're all optional, except for the comment about "resupply from" – I think it's worth considering that one a little more thoroughly. After taking a look at those, feel move this onto the next stage of review! You're doing great! :partying_face:

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 4]
```diff
@@ -2,99 +2,135 @@
...
+- :guilabel:`Address`: the address where the warehouse is located. By default, this is set to the
+  company's address in the database.
+
+.. image:: create_a_second_warehouse/create-a-second-warehouse-new-warehouse-settings.png
    :align: center
-   :alt: A filled out form for creating a new warehouse.
+   :alt: Filled out warehouse name and address settings on warehouse creation form.
+
+Under the :guilabel:`Warehouse Configuration` tab, edit the :guilabel:`Shipments` and
+:guilabel:`Resupply` settings as necessary.
```
[the remark this answers is not in the document]
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:50]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 5]
```diff
@@ -2,99 +2,135 @@
...
+   :guilabel:`Multi-Step Routes` checkbox is enabled in :menuselection:`Inventory app -->
+   Configuration --> Settings`, under the :guilabel:`Warehouse` heading. For more information about
+   routes and how they work in Odoo, see :ref:`Using Routes and Pull/Push Rules <use-routes>`.
+
+In the :guilabel:`Shipments` section, edit the following settings:
+
+- :guilabel:`Incoming Shipments`: default incoming route to follow. Choose from one, two, and
+  three-step receipts.
+- :guilabel:`Outgoing Shipments`: default outgoing route to follow. Choose from one, two, and
+  three-step deliveries.
```
[the remark this answers is not in the document]
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:63]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 6]
```diff
@@ -2,99 +2,135 @@
...
+
+In the :guilabel:`Resupply` section, edit the following settings:
+
+- :guilabel:`Dropship Subcontractors`: dropship subcontractors with components.
```
[the remark this answers is not in the document]
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:67]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst (hunk 1 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:76]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst (hunk 2 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:89]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst (hunk 3 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:99]: I understand the logic here, but I feel like the process being explained here isn't a bad thing, especially considering it could be part of the workflow for creating a second WH. I did ad a seealso block for that doc, but I'm gonna keep this section as is for now. If I receive the same feedback again, I'll consider changing it

**Comment by brse-odoo** [comment]
Hi @odoo/us-doc-review, this PR is ready for final content review. Thanks in advance!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 7]
```diff
@@ -2,99 +2,141 @@
...
+If a new warehouse is created to represent a real-world storage location with existing stock, update
+the warehouse's stock using an inventory adjustment.
+
+.. seealso::
+   :doc:`count_products`
 
-Once the form is properly configured, click on :guilabel:`Start Inventory` to be taken to the next
-page where products can be added to the inventory adjustment. Add a new product by clicking on
-:guilabel:`Create` and then fill out the product line as follows:
+Inventory can be added to a new warehouse via an inventory adjustment. Navigate to
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:97]: This first sentence seems redundant to the section's introductory sentence and doesn't transition well afterwards, I don't think. Here's a suggestion to help ease that transition into the direct action we're instructing on the UI.

```suggestion
To perform an inventory adjustment, navigate to
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 8]
```diff
@@ -2,99 +2,141 @@
...
+- :guilabel:`On Hand Quantity`: the total quantity of the product stored in the location for which
+  inventory is being adjusted. For a new warehouse or location, this should be set to `0.00`.
+- :guilabel:`UoM`: the unit of measure (UoM) used for counting the product.
+- :guilabel:`Counted Quantity`: the amount of the product being added to inventory.
+- :guilabel:`Difference`: the difference between the on-hand and counted quantities. This
+  automatically updates to reflect the value entered in the :guilabel:`Counted Quantity` column.
+- :guilabel:`Scheduled Date`: the date selected for the next inventory count for this product.
+- :guilabel:`User`: the user who recorded the inventory adjustment in the database.
+
+.. image:: create_a_second_warehouse/create-a-second-warehouse-inventory-adjustment.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:117]: This screenshot is a bit small and has both x and y axis scrolls visible. Consider retaking up to 933px width so we really capture those details for the reader, as well as at a good crop. It would also be nice to see the column header labels so we know what we're looking at.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 9]
```diff
@@ -2,99 +2,141 @@
...
-  is being adjusted; this should be zero for a new location or warehouse
-- :guilabel:`Counted`: the amount of the product that is being added to inventory
-- :guilabel:`Difference`: the difference between the *On Hand* and *Counted* values; this will
-  automatically update to reflect the value entered in the :guilabel:`Counted` column
-- :guilabel:`UoM`: the unit of measure used for counting the product
-
-.. image:: create_a_second_warehouse/product-line-configuration.png
+  identify it, if the product is tracked.
+- :guilabel:`On Hand Quantity`: the total quantity of the product stored in the location for which
+  inventory is being adjusted. For a new warehouse or location, this should be set to `0.00`.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:109]: Consider adding more context around the conditions for the value we're suggesting.
 
It's possible that someone IRL did a physical count at the new warehouse (or new ahead of time), and inputted it into the new warehouse in Odoo before we (i.e., the office admin) performed an "official" batch inventory adjustment.

```suggestion
  inventory is being adjusted. For a new warehouse or location where a count or a prior inventory
  adjustment has not taken place, this should be set to `0.00`.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 10]
```diff
@@ -2,99 +2,141 @@
...
+   If products tracked using a lot number or serial number are added without specifying the tracking
+   number, a :guilabel:`Tracked Products in Inventory Adjustment` pop-up window appears upon
+   clicking :guilabel:`Apply` on that product's line.
+
+   The pop-up indicates tracked products were added, but the lots/serial numbers were not defined.
+   To apply the adjustment without the numbers, click :guilabel:`Apply`. To add the numbers before
+   applying, click :guilabel:`Discard`, and add the numbers. Then, once ready, click
+   :guilabel:`Apply` again.
+
+   .. image:: create_a_second_warehouse/create-a-second-warehouse-popup-window.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:137]: Consider capturing the full pop-up window here, as only the right and bottom borders are visible.

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 

Nice rewrite. I like how you reordered the sections and made it more of an instructional walk-through, versus the more simple UI coverage that was there before.

Approving with comments for your consideration, which you can find below.
I'd also take a look at the weight of your compressor, and consider dialing it back some, as it's a bit strong 😁 

Can move to @samueljlieber next when you're ready, thanks!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 11]
```diff
@@ -2,99 +2,141 @@
...
+   :guilabel:`Multi-Step Routes` checkbox is enabled in :menuselection:`Inventory app -->
+   Configuration --> Settings`, under the :guilabel:`Warehouse` heading. For more information about
+   routes and how they work in Odoo, see :ref:`Using Routes and Pull/Push Rules <use-routes>`.
+
+In the :guilabel:`Shipments` section, edit the following settings:
+
+- :guilabel:`Incoming Shipments`: choose between one, two, or three-step processes for incoming
+  packages.
+- :guilabel:`Outgoing Shipments`: choose between one, two, or three-step processes for outgoing
+  packages.
```
[the remark this answers is not in the document]
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:64]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 12]
```diff
@@ -2,99 +2,141 @@
...
+
+In the :guilabel:`Resupply` section, edit the following settings:
+
+- :guilabel:`Dropship Subcontractors`: enable this feature to coordinate with vendors, and have
+  components dropshipped directly to subcontractors for manufacturing.
+- :guilabel:`Resupply Subcontractors`: allow subcontractors to be resupplied with goods from this
+  warehouse.
+- :guilabel:`Manufacture to Resupply`: when products are manufactured, they can be manufactured in
+  this warehouse.
```
[the remark this answers is not in the document]
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:73]: You are right, this is a typo - I must have looked it over when writing it. Nice catch, adjusting accordingly - Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst (hunk 7 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:97]: Nice catch, it is redundant. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst (hunk 10 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:137]: Retook screenshot and replaced with better image with all borders visible

**Comment by brse-odoo** [comment]
@samueljlieber This PR is ready for final/tech review when you have a chance, thanks!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 13]
```diff
@@ -2,99 +2,138 @@
 Create a second warehouse
 =========================
 
-A *warehouse* is a physical building or space where items are stored. In Odoo, it is possible to set
-up multiple warehouses and transfer stored items between them.
+In Odoo *Inventory*, a *warehouse* is a physical building or space where items are stored. By
+default, each Odoo database has one warehouse already pre-configured, with the address set as
+the company's address.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:7]: Early break :)
```suggestion
default, each Odoo database has one warehouse already pre-configured, with the address set as the
company's address.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst [hunk 14]
```diff
@@ -2,99 +2,138 @@
...
+In the :guilabel:`Resupply` section, edit the following settings:
+
+- :guilabel:`Dropship Subcontractors`: enable this feature to coordinate with vendors, and have
+  components dropshipped directly to subcontractors for manufacturing.
+- :guilabel:`Resupply Subcontractors`: allow subcontractors to be resupplied with goods from this
+  warehouse.
+- :guilabel:`Manufacture to Resupply`: when products are manufactured, they can be delivered to this
+  warehouse.
+- :guilabel:`Manufacture`: the route that should be followed when manufacturing goods inside
+  the warehouse. Choose from one, two, and three-step production.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:71]: another one
```suggestion
- :guilabel:`Manufacture`: the route that should be followed when manufacturing goods inside the
  warehouse. Choose from one, two, and three-step production.
```

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo! Nice job on the improvements in this PR. I found two very small nitpicks for early line breaks, as well as the commit message needs to be updated to follow the convention (it is missing the app/module). Approving now and delegating merge to you at your discretion :)

Please update the commit message (you can use `git commit --amend`):
```
[IMP] inventory: update create second warehouse doc
```
Thanks for your work!
...
@robodoo delegate=brse-odoo

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst (hunk 13 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:7]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst (hunk 14 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst:71]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
@robodoo r+
